### PR TITLE
Add updated terms of use page

### PR DESCRIPTION
**Pull request by karlchillmaid**
This PR adds a new page called `New terms of use - 2024` so users can preview our updated terms of use before they come into effect in June.

We’ll link to this page from an email, as well as from the current `Terms of use` page.

### CODE REVIEW

File: app/templates/views/terms-of-use.html [hunk 1]
```diff
@@ -11,6 +11,12 @@
 
   <h1 class="heading-large">Terms of use</h1>
 
+  <div class="govuk-inset-text">
+    <p class="govuk-body">
+      These terms will be updated on xxx. You can see the <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.new_terms_of_use') }}">new terms of use</a>.
+    </p>
```
Comment (karlchillmaid) [inline app/templates/views/terms-of-use.html:17]:
```suggestion
    <p class="govuk-body">
      We’re updating our terms of use on Monday 3 June 2024.
    </p>
    <p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.new_terms_of_use') }}">Preview the new terms</a>
    </p>
```

File: app/templates/views/terms-of-use.html [hunk 2]
```diff
@@ -11,6 +11,14 @@
 
   <h1 class="heading-large">Terms of use</h1>
 
+  <div class="govuk-inset-text">
+    <p class="govuk-body">
+      We’re updating our terms of use on Monday 3 June 2024.
+    </p>
+    <p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.new_terms_of_use') }}">Preview the new terms</a>
```
Comment (karlchillmaid) [inline app/templates/views/terms-of-use.html:18]:
```suggestion
    <p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.new_terms_of_use') }}">Read our new terms of use</a>
```

**Review by kr8n3r — APPROVED** [review]
tested locally, looks good

**Comment by idavidmcdonald** [comment]
## Inset text for the existing terms of use:

image

## New terms of use page

notify localhost_6012_new-terms-of-use

File: app/templates/views/new-terms-of-use.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  </ul>
+
+  <p class="govuk-body">
+    To help you do this, we’ve published details of <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_security') }}">Notify’s approach to security</a>.
+  </p>
+  
+  <h2 class="heading-medium" id="when-using-notify">When using Notify</h2>
+
+  <ol class="govuk-list govuk-list--number">
+    <li>Report any security breaches immediately to <a class="mailto:report@digital.cabinet-office.gov.uk">report@digital.cabinet-office.gov.uk</a>. This includes the suspected compromise of team members’ sign-in details or authentication factors, the suspected compromise of API keys and certificate warning messages.</li>
```
Comment (karlchillmaid) [inline app/templates/views/new-terms-of-use.html:59]:
```suggestion
    <li>Report any security breaches immediately to <a class="govuk-link govuk-link--no-visited-state" href="mailto:report@digital.cabinet-office.gov.uk">report@digital.cabinet-office.gov.uk</a>. This includes the suspected compromise of team members’ sign-in details or authentication factors, the suspected compromise of API keys and certificate warning messages.</li>
```

File: app/templates/views/new-terms-of-use.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    <li>Do not send unsolicited messages. Only send messages related to a transaction or something the recipient has subscribed to be updated about.</li>
+    <li>Do not send messages with abusive or offensive content.</li>
+  </ol>
+
+  <h3 class="heading-small" id="signing-in-to-notify">Signing in to Notify</h3>
+
+  <ol class="govuk-list govuk-list--number">
+    <li>You must sign in to the Notify website using a work-managed device. We recommend that, as a minimum, your device meets the requirements of the <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/files/Cyber-Essentials-Requirements-for-Infrastructure-v3-1-April-2023.pdf">NCSC’s Cyber Essentials Scheme</a>. If possible, your device should use a protective Domain Name System (DNS) service, for example the <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/information/pdns">NCSC’s PDNS</a>.</li>
```
Comment (karlchillmaid) [inline app/templates/views/new-terms-of-use.html:71]:
```suggestion
    <li>You must sign in to the Notify website using a work-managed device. We recommend that, as a minimum, your device meets the requirements of the <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/files/Cyber-Essentials-Requirements-for-Infrastructure-v3-1-April-2023.pdf">National Cyber Security Centre’s (NCSC) Cyber Essentials Scheme</a>. If possible, your device should use a protective Domain Name System (DNS) service, for example the <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/information/pdns">NCSC’s PDNS</a>.</li>
```

File: app/templates/views/new-terms-of-use.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+  <ul class="govuk-list govuk-list--bullet">
+    <li>at least once every year</li>
+    <li>when someone with access to a key leaves your team</li>
+    <li>whenever you suspect that a key may have been compromised</li>
+  </ul>
+
+  <p class="govuk-body">Do not hard-code your API keys.</p>
+
+  <p class="govuk-body">We recommend that you use a cloud service provider’s key management service to keep your API keys secure. For more information, <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/collection/developers-collection/principles">read the National Cyber Security Centre (NCSC) guidance</a>.</p>
```
Comment (karlchillmaid) [inline app/templates/views/new-terms-of-use.html:89]:
```suggestion
  <p class="govuk-body">We recommend that you use a cloud service provider’s key management service to keep your API keys secure. For more information, <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/collection/developers-collection/principles">read the NCSC guidance</a>.</p>
```

File: app/templates/views/new-terms-of-use.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    <li>Do not send unsolicited messages. Only send messages related to a transaction or something the recipient has subscribed to be updated about.</li>
+    <li>Do not send messages with abusive or offensive content.</li>
+  </ol>
+
+  <h3 class="heading-small" id="signing-in-to-notify">Signing in to Notify</h3>
+
+  <ol class="govuk-list govuk-list--number">
+    <li>You must sign in to the Notify website using a work-managed device. We recommend that, as a minimum, your device meets the requirements of the <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/files/Cyber-Essentials-Requirements-for-Infrastructure-v3-1-April-2023.pdf">National Cyber Security Centre’s (NCSC) Cyber Essentials Scheme</a>. If possible, your device should use a protective Domain Name System (DNS) service, for example the <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/information/pdns">NCSC’s PDNS</a>.</li>
```
Comment (karlchillmaid) [inline app/templates/views/new-terms-of-use.html:71]:
```suggestion
    <li>You must sign in to the Notify website using a work-managed device. We recommend that, as a minimum, your device meets the requirements of the National Cyber Security Centre (NCSC) <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/files/Cyber-Essentials-Requirements-for-Infrastructure-v3-1-April-2023.pdf">Cyber Essentials Scheme</a>. If possible, your device should use a protective Domain Name System (DNS) service, for example the <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/information/pdns">NCSC’s PDNS</a>.</li>
```

**Comment by idavidmcdonald** [comment]
New screenshot of Karl's changes

notify localhost_6012_new-terms-of-use (1)

**Review by idavidmcdonald — APPROVED** [review]
Happy with the new commits on top

**Pull request merged**